.github/workflows/test.yml

```diff
@@ -149,6 +149,59 @@ jobs:
         run: |
           ls -laR "$TEST_DIR"
 
+      - name: Run Docker Image (audit-only)
+        env:
+          FULL_IMAGE_NAME: ${{ inputs.full-image-name }}
+          WORKSPACE: ${{ github.workspace }}
+          PARAMETERS: ${{ steps.prepare.outputs.parameters }}
+          ANALYSIS_OUTPUT_DIR: ${{ steps.prepare.outputs.analysis-output-dir }}
+          BENCHMARKS_DIR: benchmarks_dir
+          PACTA_DATA_DIR: pacta-data
+          PORTFOLIO_DIR: tests/portfolios
+          REAL_ESTATE_DIR: ${{ steps.prepare.outputs.real-estate-dir }}
+          REPORT_OUTPUT_DIR: ${{ steps.prepare.outputs.report-output-dir }}
+          SCORE_CARD_DIR: ${{ steps.prepare.outputs.score-card-dir }}
+          SUMMARY_OUTPUT_DIR: ${{ steps.prepare.outputs.summary-output-dir }}
+          SURVEY_DIR: ${{ steps.prepare.outputs.survey-dir }}
+        run: |
+
+          docker run \
+          --network none \
+          --env LOG_LEVEL=TRACE \
+          --env ANALYSIS_OUTPUT_DIR="/mnt/analysis_output_dir" \
+          --env OUTPUT_DIR="/mnt/analysis_output_dir" \
+          --env BENCHMARKS_DIR="/mnt/benchmarks_dir" \
+          --env PACTA_DATA_DIR="/mnt/pacta-data" \
+          --env PORTFOLIO_DIR="/mnt/portfolios" \
+          --env REAL_ESTATE_DIR="/mnt/real_estate_dir" \
+          --env REPORT_OUTPUT_DIR="/mnt/report_output_dir" \
+          --env SCORE_CARD_DIR="/mnt/score_card_dir" \
+          --env SUMMARY_OUTPUT_DIR="/mnt/summary_output_dir" \
+          --env SURVEY_DIR="/mnt/survey_dir" \
+          --mount type=bind,readonly,source=${WORKSPACE}/${BENCHMARKS_DIR},target=/mnt/benchmarks_dir \
+          --mount type=bind,readonly,source=${WORKSPACE}/${PACTA_DATA_DIR},target=/mnt/pacta-data \
+          --mount type=bind,readonly,source=${WORKSPACE}/${PORTFOLIO_DIR},target=/mnt/portfolios \
+          --mount type=bind,readonly,source=${REAL_ESTATE_DIR},target=/mnt/real_estate_dir \
+          --mount type=bind,readonly,source=${SCORE_CARD_DIR},target=/mnt/score_card_dir \
+          --mount type=bind,readonly,source=${SURVEY_DIR},target=/mnt/survey_dir \
+          --mount type=bind,source=${ANALYSIS_OUTPUT_DIR},target=/mnt/analysis_output_dir \
+          --mount type=bind,source=${REPORT_OUTPUT_DIR},target=/mnt/report_output_dir \
+          --mount type=bind,source=${SUMMARY_OUTPUT_DIR},target=/mnt/summary_output_dir \
+          --entrypoint "/workflow.pacta.webapp/inst/extdata/scripts/run_audit.sh" \
```

Comment:
@jdhoffa @bcspragu This `entrypoint` overrride will allow the image to run in "audit-only" mode, rather than running through the entire audit and reporting process (which offers a whole bunch of places to exit with a non-`0` exit code).

Example "audit only" outputs: https://github.com/RMI-PACTA/workflow.pacta.webapp/actions/runs/12418873244/job/34673043874#step:9:13
Example "normal mode" outputs (from same JSON args): https://github.com/RMI-PACTA/workflow.pacta.webapp/actions/runs/12418873244/job/34673043874#step:11:9